README.md

```diff
@@ -32,16 +32,19 @@ let lpath = "/tempZone/home/<username>/newCollection"
 
 // After authenticating the client in an asynchronous context, any operations of the api can be done.
 async function authenticateAndCreateCollection(lpath: string) {
-    await api.authenticate()
-    let res = await api.collections.create({lpath: lpath})
     try {
+        await api.authenticate()
+        let res = await api.collections.create({lpath: lpath})
         // res.status contains the http status code of the response
         // If res.status >= 400, then res.data is null and there was an error
```

Comment:
Add period to end of each comment/statement.

The comments for the `IrodsHttpClient` constructor need periods.
The comment for `let lpath = ...` needs a period too.

Reply:
done